### PR TITLE
[DOCS-6669] - Add restore release option

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Comment by Artur-Piechuta** [comment]
1. "You can restore your system to a previous release (...)"
It should be _project_ and not _system_, because projects are version by releases.
2. I think that more relevant place for this information is here:
image

File: process-automation/latest/model/projects.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ To release a project:
 
 Selecting the **View Releases** option for a project displays the version history for the project.
 
+You can restore your project to a previous release by selecting the three dots to the right of the release you want to restore, and then selecting **Restore this Release**.
```
Comment (Artur-Piechuta) [inline process-automation/latest/model/projects.md:68]:
```suggestion
We can restore your project to a previous release by selecting the three dots to the right of the release you want to restore, and then selecting **Restore this Release**.
```

**Review by Artur-Piechuta — CHANGES_REQUESTED** [review]

File: process-automation/latest/model/projects.md (hunk 1 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/projects.md:68]: I wont make those changes cheers. We use the You  instead here.

**Review by Artur-Piechuta — APPROVED** [review]
LGTM

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**